processor/spanmetricsprocessor/config.go

```diff
@@ -79,3 +80,49 @@ func (c Config) GetAggregationTemporality() pmetric.AggregationTemporality {
 	}
 	return pmetric.AggregationTemporalityCumulative
 }
+
```

Comment:
nit: What do you think about adding a compile time check for the interface implementation to make sure this doesn't break again?

```
var _ component.ConfigValidator = (Config)(nil)
```